crypto/cryptlib.c

```diff
@@ -401,8 +401,11 @@ void *OPENSSL_stderr(void)	{ return stderr; }
 int CRYPTO_memcmp(const void *in_a, const void *in_b, size_t len)
```

Comment:
The prototype needs to read:

    int CRYPTO_memcmp(const volatile void *in_a, const volatile void *in_b, size_t len)

in order to defeat possibly very aggressive LTO.